applications/tasks/src/kz_account_crawler.erl

```diff
@@ -326,7 +326,13 @@ maybe_test_for_low_balance(AccountId, AccountJObj) ->
 
 -spec test_for_low_balance(ne_binary(), kz_account:doc()) -> 'ok'.
 test_for_low_balance(AccountId, AccountJObj) ->
-    CurrentBalance = wht_util:current_balance(AccountId),
+    CurrentBalance = case wht_util:current_balance(AccountId) of
+                         Balance when Balance == 0 ->
+                             kazoo_modb_maintenance:fix_rollup(AccountId),
+                             timer:sleep(5000),
```

Comment:
With changes in #3346 I don't think this is necessary here. Also it's making crawler slower.